js/src/Ice/Logger.js

```diff
@@ -1,7 +1,5 @@
 // Copyright (c) ZeroC, Inc.
 
-import fs from "fs";
```

Comment:
We no longer import Node.js specific modules in files that are used by both Node/Browser. FileLogger used "fs" for Node and was split in FileLoger.js (Node implementation), FileLoger.browser.js (mockup for browser)